src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryEndpoint.java

```diff
@@ -254,7 +255,7 @@ public boolean equals(Object obj) {
         return true;
     }
     
-    @Extension
+    @Extension @Symbol("dockerHubEndpoint")
```

Comment:
:bug: not necessarily for Hub (in fact usually not), and anyway this class is not `abstract` so the actual symbol should be as short as possible: say, `endpoint`.